src/main/java/ru/sbtqa/tag/pagefactory/stepdefs/SetupStepDefs.java

```diff
@@ -113,11 +113,13 @@ public void setUp() {
 
     @After
     public void tearDown() {
-        if (VideoRecorder.getInstance().isVideoStarted()) {
-            String videoPath = VideoRecorder.getInstance().stopRecording();
-            if (videoPath != null) {
-                ParamsHelper.addVideoParameter(VideoRecorder.getInstance().getVideoPath());
-                VideoRecorder.getInstance().resetVideoRecorder();
+        if (PageFactory.videoRecorderIsEnabled()) {
+            if (VideoRecorder.getInstance().isVideoStarted()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1066)